03.macos/05.solving-problems/08.protect-mail-activity/docs.en.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Protect Mail Activity and AdGuard'
+taxonomy:
+    category:
+        - docs
+---
+
+## In a nutshell
+
+Apple's Mail app now uses a proxy to hide a user's IP Address when downloading images from emails.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/MAC/mac_protectMailActivity.png"> 
+
+
+However, it won't work if there's an active VPN connection. As it treats AdGuard as a VPN, it won't preload images automatically.
+
+Apple explains this issue [here](https://support.apple.com/en-us/HT212797).
+
+## In detail
+
+AdGuard for Mac now uses macOS built-in socket filtering based on the network extensions API. This new and rather buggy mechanism replaced good old Kernel extensions. Over the last 1.5 years, we've reported more than 20(!) bugs to Apple regarding their new filtering method.
+
+The network extensions API has a VPN-like configuration with a list of route-like entries.
+On Big Sur, AdGuard developed "split-tunnel" rules to avoid creating the "default route" rule because it causes problems on early Big Sur releases.
```

Comment:
```suggestion
On Big Sur, AdGuard used "split-tunnel" rules to avoid creating the "default route" rule because it causes problems on early Big Sur releases. These problems were solved in Monterey so nothing prevents us from using the "default route" rule.
```